README.md

```diff
@@ -25,14 +25,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
 
 <!-- TABLE OF CONTENTS -->
 
```

Comment:
- [ ] Kindly ensure that your Logo design sticks as much as possible to the one from Figma design as required fro the projec, below you can see how your design looks for now 
![Screenshot from 2023-02-16 01-17-50](https://user-images.githubusercontent.com/83042742/219214496-4d55c710-f9f2-462f-966b-9a31851c4e10.png)

Reply:
done